src/main/java/views/AcesUp/SpanAcesUp.html

```diff
@@ -228,7 +228,7 @@
 $("#dealButton").click(function(){
 $.ajax({
   type: "POST",
-  url: "/dealGame",
+  url: "/dealSpanGame",
```

Comment:
I'm glad you guys found a fix to the annoying error we kept getting. Looking at it now, separating the URL's for the Spanish and English games makes a lot of sense.